StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1011ClosingSquareBracketsMustBeSpacedCorrectly.cs

```diff
@@ -83,26 +84,12 @@ private void HandleCloseBracketToken(SyntaxTreeAnalysisContext context, SyntaxTo
                 return;
             }
 
-            bool precededBySpace;
-            bool firstInLine;
-
-            bool followedBySpace;
-            bool lastInLine;
+            bool firstInLine = token.IsFirstInLine();
+            bool precededBySpace = firstInLine ? true : token.IsPrecededBySpace();
```

Comment:
You could also write it like `precededBySpace = firstInLine || token.IsPrecededBySpace();`